public/app/plugins/datasource/prometheus/querybuilder/shared/LabelFilterItem.tsx

```diff
@@ -98,7 +99,7 @@ export function LabelFilterItem({
             aria-label={selectors.components.QueryBuilder.matchOperatorSelect}
             value={toOption(item.op ?? defaultOp)}
             options={operators}
```

Comment:
```suggestion
            options={[
  lokiOperators.equals,
  lokiOperators.doesNotEqual,
  lokiOperators.matchesRegex,
  lokiOperators.doesNotMatchRegex,
]}
```

Reply:
this is also used on `line 45` so i think it makes sense to keep it here and have them in sync.